### DISCUSSION TITLE
the community version of graal be compatible with my open jdk8?

### DISCUSSION THREAD

**Discussion by barry-gq** [oracle/graal#4826] [Q&A] [answered]
The graalVM official website mentions that the community version supports at least openjdk 11, but my open jdk version is 8. 
Can I replace the java version in the graalvm bin directory for use?
If so, how can I replace it?

**Comment by alina-yur** [comment, marked as answer]
HI @barry-gq, you would need to use GraalVM builds — GraalVM has a set of additional components that are integrated with each other, so it wouldn't work like that. Please give our JDK 11 builds a try, it should work, and if not, please report the results to us.
